splinepy/multipatch.py

```diff
@@ -144,7 +144,7 @@ def boundaries(self):
         max_BID = self.interfaces.min()
 
         boundary_list = []
-        for i_bid in range(-1, max_BID, -1):
+        for i_bid in range(-1, max_BID - 1, -1):
```

Comment:
I know you wrote in doc but can you also add comment at l.143 above that those are negative, hence, searching for min()